src/issues/components/ConflictSetupTitle.vue

```diff
@@ -0,0 +1,6 @@
+<template>
+  <h1>Start a bloody conflict</h1>
```

Comment:
should be translatable

Reply:
Yeah, and should be a text we actually want to have in there...^^